sdks/python/apache_beam/options/pipeline_options_validator_test.py

```diff
@@ -329,7 +330,11 @@ def get_validator(matcher):
     ]
 
     for case in test_case:
-      errors = get_validator(case['on_success_matcher']).validate()
+      matcher = case['on_success_matcher']
+      if matcher and sys.version_info[0] >= 3 and type(matcher) is bytes:
```

Comment:
Do we need the version check here?
I think we can only check for bytes and decode on both Python 2 and 3.

Reply:
I was not sure but let me try